### DISCUSSION TITLE
Datagrid Detail Pane

### DISCUSSION THREAD

**Discussion by KishanRavindran** [vmware-clarity/ng-clarity#121] [Q&A]
Hi All,
      I am using the clr-datagrid and have used detail Pane to showcase a object details in a row. But the object will not be there for certain Json. So in that I don't want to show the detail Pane. How can I not show the detail Pane when the object is not there.

Thanks in Advance.

**Comment by ashleyryan** [comment]
Can you put together a stackblitz example with your issue: https://stackblitz.com/@clr-team/
